integration/testdata/rails/config/puma.rb

```diff
@@ -22,7 +22,7 @@
 environment ENV.fetch("RAILS_ENV") { "development" }
 
 # Specifies the `pidfile` that Puma will use.
-pidfile ENV.fetch("PIDFILE") { "tmp/pids/server.pid" }
+pidfile ENV.fetch("PIDFILE") { "/tmp/pumd.pid" }
```

Comment:
@ryanmoran this specifically had to change, without this line, the app failed to start on Jammy due to permissions errors trying to write to `/workspace/tmp/pids`.

Reply:
This makes sense. It probably got rolled back when I upgraded the app to Rails 6.1 and the tool regenerated a bunch of these files.